CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.78.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.77.0...3.78.0) - 2020-10-22
```

Comment:
You should update the release date.

Reply:
Done. (Do not pay attention to the fact there is two commit ; I'll do a squash merge)